source/_integrations/sensibo.markdown

```diff
@@ -45,127 +45,211 @@ then in the Sensibo app log you will be able to distinguish between actions
 done in the app and actions done by Home Assistant.
 {% endtip %}
 
+## Supported device
+
+The **Sensibo** {% term integration %} supports the following devices and accessories.
+
+- **Sensibo Sky**: Smart AC control device.
+- **Sensibo Air**: Smart AC control device.
+- **Sensibo Air Pro**: Smart AC control device with air quality monitoring.
+- **Sensibo Pure**: Smart air purifier.
+- **Sensibo Elements**: Smart air quality monitoring.
+- **Sensibo Room Sensor**: Motion sensor and temperature readings (needs to be connected with an Air device).
+
 {% include integrations/config_flow.md %}
 
-## Binary sensors
+{% configuration_basic %}
+API key:
+  description: The previously created API key.
+{% endconfiguration_basic %}
+
+## Data fetching and limitations
+
+Data is polled from the **Sensibo** API once every minute for all devices.
+
+If polling cannot happen because of no connectivity or a malfunctioning API, it will retry a few times before failing.
+The user can use the [`homeassistant.update_entity`](homeassistant#action-homeassistantupdate_entity) action to manually try again later, in the case the user has solved the connectivity issue.
+
+## Troubleshooting
+
+This service is reliant on an internet connection and that the **Sensibo** API is available. Here are the things you can try before raising an issue:
+
+- Check that internet is available in your Home Assistant instance.
+- Check that the **Sensibo** API is available by clicking [here](https://home.sensibo.com/api/v1/users/me). If you have previously logged in to Sensibo web, you will get a JSON back with the provided information about your account. If not logged in, the API will respond with `login_required`.
```

Comment:
```suggestion
- Check that the **Sensibo** API is available by accessing the [Sensibo API page](https://home.sensibo.com/api/v1/users/me). If you have previously logged in to Sensibo web, you will get a JSON back with the provided information about your account. If not logged in, the API will respond with `login_required`.
```
Instead of "here" I would describe where the link is going. Please correct the text if I got that wrong.